src/schema/schema.rs

```diff
@@ -144,6 +145,28 @@ impl SchemaBuilder {
         self.add_field(field_entry)
     }
 
+    /// Adds a ip field.
+    /// Returns the associated field handle
+    /// Internally, Tantivy simply stores ips as u64,
+    /// while the user supplies IpAddr values for convenience.
+    ///
+    /// # Caution
+    ///
+    /// Appending two fields with the same name
+    /// will result in the shadowing of the first
+    /// by the second one.
+    /// The first field will get a field id
+    /// but only the second one will be indexed
```

Comment:
This is an excellent point, but it is very strange to put it here only (!?)

Can you create an issue to fix this.
I think we should panic instead, and document that this is forbidden.